### ISSUE TITLE
Laika hang when in a hidden directory

### ISSUE THREAD

**Issue by boekkooi** [arunoda/laika#39]
executing laika in a hidden directory (for example '.laika') will make it hang on `loading initial app pool...`.
It looks like the meteor server is not starting correctly.

Recreate using:

```
git clone https://github.com/arunoda/hello-laika.git .hello
cd .hello
laika
```

**Comment by arunoda** [comment]
I tested this today. Yes it is not working.
The problem is when the meteor is inside a hidden directory it does not pick injected laika server code.

And also seems like it does not load any js at all.

Is your app running properly inside a hidden folder?

**Comment by arunoda** [comment]
Related meteor issue seems to be get fixed with `0.6.5`. Can you please update your project and try again.
